blocks/init/src/Blocks/custom/tabs/assets/tabber.js

```diff
@@ -0,0 +1,30 @@
+export class Tabber {
+    constructor(options) {
+        this.tabsElement = options.tabsElement;
+        this.options = options;
+        this.tabsElement.querySelector(`.${this.options.tabJsClass}`).classList.add(this.options.tabVisibleClass);
+        this.tabPanel = this.tabsElement.querySelector(`.${this.options.tabPanelJsClass}`); 
+    }
+    tabPanelClickHandler(e){
+        this.tabsElement.querySelector(`.${this.options.tabVisibleClass}`).classList.remove(this.options.tabVisibleClass);
```

Comment:
Also, the classes denoting the visibility or some action (`is-active` or `has-color`, etc.) are also defined in the constructor body like

```js
constructor(options) {
	this.tabsElement = options.tabsElement;
	...
	this.IS_VISIBLE_CLASS = 'is-visible';
}
```

Then you can use that when setting the classes or removing them

```js
        this.tabPanel.querySelector(`.${this.IS_ACTIVE_CLASS}`).classList.remove(this.IS_ACTIVE_CLASS);
```